www_src/pages/element/element.jsx

```diff
@@ -81,17 +80,28 @@ render(React.createClass({
   load: function () {
     api({uri: this.uri()}, (err, data) => {
       if (err) {
-        return console.error('Error loading element', err);
+        return this.recordError('Error loading element', err);
       }
-
       if (!data || !data.element) {
-        return console.log('No element found');
+        return this.recordError('No element found');
       }
-
       this.setState({element: data.element});
-
     });
   },
+
+  testSnackbar: function() {
+    this.recordError("Oh dear someone pressed the button");
+  },
+
```

Comment:
The `recordError` function is probably going to get put in its own mixin, along with a `generateSnackbar()` function so that any component can include the mixin without needing to require separate components etc.